aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -152,6 +152,11 @@ public static CreateDbInstanceReadReplicaRequest createDbInstanceReadReplicaRequ
             builder.storageThroughput(model.getStorageThroughput());
             builder.storageType(model.getStorageType());
         }
+
+        if (ResourceModelHelper.isMySQL(model)) {
+            builder.dbParameterGroupName(model.getDBParameterGroupName());
```

Comment:
If dbparameter group is set unconditionally, what effect would that have on non-mysql engines?